### ISSUE TITLE
[Model Request] Please release the curriculum-trained final model used in the paper

### ISSUE THREAD

**Issue by lvy-ava** [NVlabs/EARLI#3]
## Summary

I am reproducing the results of the paper "Accelerating Vehicle Routing via AI-Initialized Genetic Algorithms" (EARLI). I have successfully run the full pipeline (RL inference -> HGS local search -> cuOpt injection) on 5 benchmark sets (SP 500, Rio 500, Synthetic 500, SP 100, SP 200), but the absolute numbers differ from the paper. After extensive debugging, I believe the root cause is that the publicly available model `model_vrp_500_sp.m` is **not** the final curriculum-trained model `vrp_500.m` used in the paper.

## What I have done

1. **Cloned the official repo** and set up the environment (WSL Ubuntu + cuOpt 25.05.01 + hygese).
2. **Ran RL inference** with `config_test.yaml` parameters (n_beams=8, temperature=0.3, agg_type=sum, lazy=True, deterministic_test_beam=True) - all matching the official config.
3. **Ran cuOpt injection** with time budgets [1s, 6s, 20s] on 256 test instances per dataset.
4. **Verified all 5 experiments**: feasibility 100%, vehicle counts match between cuOpt and cuOpt_RL, RL injection consistently reduces cost gap.

## Reproduced results vs Paper (SP 500, cuOpt)

| Metric | Paper | My reproduction |
|--------|-------|-----------------|
| cuOpt default 1s gap | ~8.5% | 9.41% |
| cuOpt + EARLI 1s gap | ~4.5% | 8.76% |
| Improvement at 1s | **4.0 pp** | **0.65 pp** |
| Win rate (RL better) | ~95% | 64.5% |
| RL t0 (init) gap | ~7% | 11.4% |

## Debugging done (all ruled out)

I systematically ruled out the following potential causes:

1. **Inference config mismatch** - Tested 4 parameter combinations (n_beams=1/8, temperature=0.3/1, agg_type=sum/mean, lazy=True/False) on 10 instances. n_beams=8 is correct.
2. **vehicle_penalty / unused_capacity_penalty** - Tested 0 vs 0.5/0.1. Identical solution costs; only reward values differ.
3. **Train/test distribution mismatch** - Training set and test set are nearly identical (radius 49952 vs 49954).
4. **Local search insufficient** - HGS local search converges in ~0.1s per instance.
5. **Distance matrix scaling** - Scaling (1e3/radius) is required and correctly applied.

## Root cause

The publicly available model is `model_vrp_500_sp.m` (directly trained). However, the paper states the final model was trained via **curriculum learning**:

```
Phase 1: size 50,  4000 epochs -> vrp_50.m
Phase 2: size 100, 1000 epochs -> vrp_100.m  (pretrained: vrp_50.m)
Phase 3: size 200,  800 epochs -> vrp_200.m  (pretrained: vrp_100.m)
Phase 4: size 500,  300 epochs -> vrp_500.m  (pretrained: vrp_200.m)
```

The `earli/pretrained_models/` directory only contains `model_vrp_{100,500}_{sp,L2}.m`, which appear to be directly trained models, not the curriculum-trained final models.

## Request

Could you please release:

1. The **curriculum-trained final model** (`vrp_500.m` or equivalent) used to produce the paper's results.
2. Optionally, the intermediate curriculum checkpoints (`vrp_50.m`, `vrp_100.m`, `vrp_200.m`).
3. The exact `config_test.yaml` / `config_initialization.yaml` / `config_cuopt.yaml` used for the paper's experiments.

This would greatly help the community reproduce the paper's results. Thank you for your work and for open-sourcing the code!

## Environment

- OS: Windows 11 + WSL2 (Ubuntu 22.04)
- GPU: NVIDIA RTX 3070 8GB
- cuOpt: 25.05.01
- PyTorch: 2.13.0+cu130
- hygese: local dev build
- Python: 3.11
